backup/moodle2/backup_local_annoto_plugin.class.php

```diff
@@ -0,0 +1,77 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * Annoto backup class.
+ *
+ * @package    local_annoto
+ * @subpackage annoto
+ * @copyright  Annoto Ltd.
+ * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
+ */
+class backup_local_annoto_plugin extends backup_local_plugin {
+    /**
+     * Returns the backup configration of annoto data for particular module.
+     */
+    public function define_module_plugin_structure() {
+        // Flag that determines that user data is included in the backup.
+        $userinfo = $this->get_setting_value('userinfo');
+        $plugin = $this->get_plugin_element(null, null, null);
+
+        $completion = new backup_nested_element(
+            'local_annoto_completion',
+            ['id'],
+            [
+                'cmid',
+                'enabled',
+                'totalview',
+                'comments',
+                'replies',
+                'completionexpected',
+                'usermodified',
```

Comment:
According to the quiz reference, no usermodified is unset and never used on restore